spec/services/listing_service/api/shapes_spec.rb

```diff
@@ -26,6 +26,7 @@
               { locale: "en", name: "Selling", action_button_label: "Buy" },
               { locale: "fi", name: "Myydään", action_button_label: "Osta" }
             ],
+            url_source: "Selling",
```

Comment:
You could also add `url` to Shape entity and test:

`expect(shape[:url]).to eql("selling")`

Reference to `TransactionType` model will be removed from this spec in the future.